nilearn/surface/tests/test_surface.py

```diff
@@ -594,21 +599,38 @@ def test_vol_to_surf(kind, n_scans, use_mask):
         mask_img = None
     if n_scans == 1:
         img = image.new_img_like(img, image.get_data(img).squeeze())
+
     fsaverage = datasets.fetch_surf_fsaverage()
+
     mesh = load_surf_mesh(fsaverage["pial_left"])
     inner_mesh = load_surf_mesh(fsaverage["white_left"])
     center_mesh = (
         np.mean([mesh.coordinates, inner_mesh.coordinates], axis=0),
         mesh.faces,
     )
+
     proj = vol_to_surf(
         img, mesh, kind="depth", inner_mesh=inner_mesh, mask_img=mask_img
     )
     other_proj = vol_to_surf(img, center_mesh, kind=kind, mask_img=mask_img)
+
     correlation = pearsonr(proj.ravel(), other_proj.ravel())[0]
+
     assert correlation > 0.99
-    with pytest.raises(ValueError, match=".*interpolation.*"):
-        vol_to_surf(img, mesh, interpolation="bad")
+
+
+def test_vol_to_surf_labels(img_labels, single_mesh):
```

Comment:
This test fails with current main branch ?

Reply:
Will check!

Reply:
Ok, it returns the same output on the main as on this branch. 
The projected surface only has the background label, which makes me think that our fixtures are not suitable for the test.

Reply:
> it returns the same output on the main as on this branch.

This wasn't true because I was not using the implementation on main and the test actually failed when I did.

But the output on this branch was still just the background label. So I am using a better mesh now.